src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -64,26 +85,68 @@ export default class ProjectMetrics extends React.Component<IProjectMetricsProps
         );
     }
 
+    private refresh = () => {
+        this.forceUpdate();
+    }
+
+    private buildValue(hoveredCell) {
+        const { radius, angle, angle0 } = hoveredCell;
+        const truedAngle = (angle + angle0) / 2;
+        return {
+            x: radius * Math.cos(truedAngle),
+            y: radius * Math.sin(truedAngle),
+        };
+    }
+
     private renderMetrics() {
         const sourceAssetCount = this.getSourceAssetCount();
-        const visitedAssetCount = this.getVisitedAssetsCount();
         const taggedAssetCount = this.getTaggedAssetCount();
-        const nonVistedAssetCount = sourceAssetCount - this.state.projectAssetsMetadata.length;
-
-        const assetChartData = [
-            {
-                angle: visitedAssetCount,
-                label: interpolate(strings.projectMetrics.visitedAssets, { count: visitedAssetCount }),
-            },
-            {
-                angle: taggedAssetCount,
-                label: interpolate(strings.projectMetrics.taggedAssets, { count: taggedAssetCount }),
-            },
-            {
-                angle: nonVistedAssetCount,
-                label: interpolate(strings.projectMetrics.nonVisitedAssets, { count: nonVistedAssetCount }),
-            },
-        ];
+        const visitedAssetCount = this.getVisitedAssetsCount();
+        const assetChartSize = window.innerWidth >= 1920 ? 250 : 200;
```

Comment:
do we use this number (1920) anywhere else? Seems like a good const candidate, but perhaps this is the only usage. Often time 'magic numbers' can be hard to track down later :)

Reply:
Good catch - this is the only place where we are doing something like this right now.  We can definitely move out to a constant though.